app/controllers/items_controller.rb

```diff
@@ -373,15 +437,18 @@ def self.normalize_source_id(new_source_id)
   def source_id
     new_id = ItemsController.normalize_source_id(params[:new_id])
     @object.set_source_id(new_id)
-    # TODO: the content= and content_will_change! calls belong in dor-services, for this method and other similar methods.
-    # can then clean up allow(idmd).to receive(:"ng_xml") (and "content_will_change!") in tests.  getting rid of "Metadata\.content\s*=\s*"
+    # TODO: the content= and content_will_change! calls belong in dor-services,
+    # for this method and other similar methods. can then clean up
+    # allow(idmd).to receive(:"ng_xml") (and "content_will_change!") in tests.
+    # getting rid of "Metadata\.content\s*=\s*"
```

Comment:
Well, as you can see, one set of lines fits in this github view and one doesn't.  Guess which one fits and can be read in it's entirety without scrolling the view?